css/index.css

```diff
@@ -1,3 +1,476 @@
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+    max-width: 100%;
+    margin: 0 auto;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/******************** HOME PAGE CODE ********************/
+
+.navBar{
+	width: 880px;
+	display:flex;
+	align-items: center;
+	justify-content: space-between;
+	margin-bottom: 50px;
+}
+
+nav{
+	display:flex;
+	width: 610px;
+	justify-content: space-between;
+}
+
+nav a{
+	text-decoration: none;
+	color:grey;
+}
+
+.eyeCatcher{
+	width:880px;
+	display:flex;
+	justify-content: flex-end;
+}
+
+.slogan{
+	font-size: 90px;
+	text-align: center;
+	display: flex;
+	flex-wrap: wrap;
+	align-content: center;
+	justify-content: center;
+}
+
+.button{
+	border: 1.5px solid black;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 18px;
+	padding: 10px 55px;
+	text-decoration: none;
+	color: black;
+}
+
+.content{
+	border-top: 1.5px solid black;
+	border-bottom: 1.5px solid black;
+	width:880px;
+	margin-top:50px;
+	margin-bottom: 50px;
+}
+
+.first{
+	width:880px;
+	display: flex;
+	margin-top:50px;
+	margin-bottom: 45px;
+}
+
+.featurescontent, .aboutcontent{
+	width:50%;
+}
+
+.middle-img{
+	width: 880px;
+}
+
+.second{
+	width:880px;
+	display: flex;
+	justify-content: space-between;
+	margin-top:30px;
+	margin-bottom: 50px;
+}
+
+.servicescontent, .productcontent, .visioncontent{
+	width:32%;
+}
+
+address{
+	display: flex;
+	flex-direction: column;
+}
+
+.number, .email{
+	margin-top:15px;
+}
+
+.copyright{
+	margin-top: 50px;
+	margin-left:370px;
+}
+
+body{	
+	margin-left:20%;
+}
+
+/******************** SERVICES PAGE CODE ********************/
+
+/* Webpage Sizing
+	.box{
+		position:relative;
+		border: 5px solid red;
+		height:210px;
+		width:880px;
+		bottom:212px;
+}*/
+h4{
+	font-size: 25px;
+}
+
+h2{
+	font-size: 18px;
+}
+
+.allContent{
+	border-top: 1.5px solid black;
+	border-bottom: 1.5px solid black;
+	width:880px;
+	height: 850px;
+	padding-top:35px;
+	margin:35px 0px;
+	display:flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+
+.contentS{
+	border: 1.5px solid black;
+	height:240px;
```

Comment:
Try not to use the height property too much with responsive design. As a page gets smaller and the parent element has less space for its children, the child elements might poke out of the parent container if its height isn't flexible.

Instead of setting a height property, try adding some margin on the top and bottom of these cards to space them out '' '